### PR TITLE
Show source path above the read-only preview editor

### LINKED ISSUE #253: [UX] Show Source Path
Given that the source may not point to the active file, I suggest adding a file path to the source heading. See attached image
Debugger source

### PR DESCRIPTION
**Pull request by jtpio**
Fixes #253.

Show the source path as we step into the code, so it's easier to know where the current execution has stopped.

show-source-path

### CODE REVIEW

**Comment by jtpio** [comment]
This does not remove the `uppercase` from the titles in the headers, as this could be addressed separately.

File: src/callstack/index.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ class CallstackHeader extends Widget {
     super({ node: document.createElement('header') });
 
     const layout = new PanelLayout();
-    const span = new Widget({ node: document.createElement('span') });
+    const span = new Widget({ node: document.createElement('h2') });
```
Comment (afshin) [inline src/callstack/index.ts]: Maybe this should be called `title` like the other one.

File: src/variables/index.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ class VariablesHeader extends Widget {
   constructor(title: string) {
     super({ node: document.createElement('header') });
     const layout = new PanelLayout();
-    const span = new Widget({ node: document.createElement('span') });
+    const span = new Widget({ node: document.createElement('h2') });
```
Comment (afshin) [inline src/variables/index.ts]: Also here...

File: src/callstack/index.ts (hunk 1 above)
Comment (jtpio) [reply src/callstack/index.ts]: Yes that's better.

**Pull request merged**